automation/test-windows.sh

```diff
@@ -103,85 +67,49 @@ delete_vm(){
   local template_name=$2
   set +e
   #stop vm
-  ./virtctl --kubeconfig=$kubeconfig stop $vm_name
+  ./virtctl stop $vm_name -n $namespace
   #delete vm
-  _oc process -o json $template_name NAME=$vm_name PVCNAME=disk-win | \
-  _oc delete -f -
+  oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt| \
+  oc delete -f -
   set -e
 }
 
 run_vm(){
   vm_name=$1
-  template_path="../../dist/templates/$vm_name.yaml"
-  local template_name=$( _oc get -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers )
+  template_path="dist/templates/$vm_name.yaml"
+  local template_name=$( oc get -n ${namespace} -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers -n kubevirt )
   running=false
 
   #If first try fails, it tries 2 more time to run it, before it fails whole test
   for i in `seq 1 3`; do
     error=false
 
-    # windows 2019 doesn't support rtc timer. 
-    if [[ $TARGET =~ windows2019.* ]]; then
-      _oc process -o json $template_name NAME=$vm_name PVCNAME=disk-win | \
-      jq '.items[0].spec.template.spec.volumes[0]+= {"ephemeral": {"persistentVolumeClaim": {"claimName": "disk-win"}}} | 
-      del(.items[0].spec.template.spec.volumes[0].persistentVolumeClaim) | del(.items[0].spec.template.spec.domain.clock.timer.rtc) |
-      .items[0].metadata.labels["vm.kubevirt.io/template.namespace"]="default"' | \
-      _oc apply -f -
-    else
-      _oc process -o json $template_name NAME=$vm_name PVCNAME=disk-win | \
-      jq '.items[0].spec.template.spec.volumes[0]+= {"ephemeral": {"persistentVolumeClaim": {"claimName": "disk-win"}}} | 
-      del(.items[0].spec.template.spec.volumes[0].persistentVolumeClaim) |
-      .items[0].metadata.labels["vm.kubevirt.io/template.namespace"]="default"' | \
-      _oc apply -f -
-    fi
-
-    validator_pods=($(_oc get pods -n kubevirt -l kubevirt.io=virt-template-validator -ocustom-columns=name:metadata.name --no-headers))
-
-    for pod in ${validator_pods[@]}; do
-      _oc logs -n kubevirt $pod
-    done
-
+    oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
+    jq 'del(.items[0].spec.dataVolumeTemplates[0].spec.pvc.accessModes) |
+    .items[0].spec.dataVolumeTemplates[0].spec.pvc+= {"accessModes": ["ReadWriteOnce"]} | 
+    .items[0].metadata.labels["vm.kubevirt.io/template.namespace"]="kubevirt"' | \
+    oc apply -f -
+    
     # start vm
-    ./virtctl --kubeconfig=$kubeconfig start $vm_name
+    ./virtctl start $vm_name -n $namespace
+    sleep 10
+    oc wait --for=condition=Ready --timeout=${timeout}s dv/$vm_name -n $namespace
```

Comment:
I think I saw one of the CI lanes fail here because the DV was not created yet. 
You can drop this `oc wait` because it is guaranteed that a VMI will not start until the DV that was assigned to it is ready, so `oc wait`ing for the `vm` condition is enough